lst_scan_joint_values:
  - [ -58,  -87,  -81,   0,  60,  -55]
  - [ -74, -119, -130,  79,  76,  -52]
  - [ -90,  -87,  -81,  -2,  92,  -49]
  - [-106, -123, -120,  74, 108,  -46]
  - [-118, -102,  -65,  -2, 120,  -44]
  - [-118, -128, -109,  69, 120,  -44]
  - [-107,  -93,  -75,  -1, 107,  -47]
  - [ -90, -120, -127,  77,  93,  -49]
  - [ -74,  -85,  -83,  -1,  76,  -52]
  - [ -58, -120, -128,  79,  61,  -55]